tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Please add `declaration ` and `declarationDir`.

Reply:
Yes, I did that in the recent commit.